desktop_version/src/XMLUtils.cpp

```diff
@@ -0,0 +1,142 @@
+#include <SDL.h>
+#include <tinyxml2.h>
+
+namespace xml
+{
+
+// Helper functions for these utils (not exported)
+
+tinyxml2::XMLDocument& get_document(tinyxml2::XMLNode* parent)
```

Comment:
May be worth setting this to static (maybe inline too?).

Reply:
What would be the reasoning behind that?

Reply:
Since it's not meant to be exported it would reduce the scope and prevent it from being exported unintentionally, while almost making it easier for the compiler to inline it.

Reply:
Makes sense to me. Added, rebased, and force-pushed.